src/shared/models/__tests__/TopicModel.test.js

```diff
@@ -0,0 +1,33 @@
+import TopicModel from '../TopicModel';
+
+describe('TopicModel', () => {
+    
+    describe('objet creation from backend data', () => {
```

Comment:
Last time I saw this typo in 2015. It was made by a French developer as `objet` is an object in French. Do you know French?

Reply:
No :( 
fixed in [https://github.com/warsawjs/topics-manager/pull/8/commits/47823e3cd8ce749d9c0b7260a9eefd564b877fd2]